.github/workflows/tests.yml

```diff
@@ -26,6 +26,8 @@ jobs:
             symfony-version: 6.0.*
           - php-version: 8.1
             symfony-version: 6.0.*
```

Comment:
I'm not sure I want to run tests on 6.0 if we are running 6.1
In fact, all 6.0 jobs in this file should be moved on 6.1